easybuild/easyconfigs/c/ChIP-Seq/ChIP-Seq-1.5-1-goolf-1.4.10.eb

```diff
@@ -15,7 +15,7 @@ description = """The ChIP-Seq software provides methods for the analysis of ChIP
 
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 
-source_urls = ['http://downloads.sourceforge.net/project/chip-seq/chip-seq/%(version)s/']
+source_urls = ['http://downloads.sourceforge.net/project/chip-seq/chip-seq/1.5.1/']
```

Comment:
@migueldiascosta please avoid hardcoding stuff like this, since we're bound to miss this later

```python
source_urls = ['http://downloads.sourceforge.net/project/chip-seq/chip-seq/%s/' % '.'.join(version.split('-'))]
```

Any idea why we're using `1.5-1` as version? Is that a mistake?